tests/infra/remote.py

```diff
@@ -60,15 +61,22 @@ def log_errors(out_path, err_path):
     error_filter = ["fail", "fatal"]
     try:
         errors = 0
+        tail_lines = deque(maxlen=15)
```

Comment:
why maxlen=15?

Reply:
This is the N in #303, how many lines we store. `deque` with a `maxlen` creates a cheap ringbuffer. Do you think we should be storing more lines? Or fewer?

Reply:
This is the N in #303, how many lines we store. `deque` with a `maxlen` creates a cheap ringbuffer. Do you think we should be storing more lines? Or fewer?

Reply:
ah ok so if it overflows it replaces lines, so you get the last 15 of the file. No, 15 seems reasonable, at least for now :)